event_rest_api/pydantic_models/event_registration_param.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from typing import List
+
+from odoo.addons.pydantic import models
+
+
+class EventRegistrationParam(models.BaseModel):
+
+    firstname: str
+    lastname: str
+    email: str
+    phone: str = None
+    event_ticket_id: int = None
+
+
+class EventRegistrationParamList(models.BaseModel):
```

Comment:
@qgroulard IMO This class should not exists.... Do  you really need to nest a list as attribute of an other model?